Kitodo/src/main/java/de/sub/goobi/config/ConfigCore.java

```diff
@@ -147,4 +151,22 @@ public static Duration getDurationParameter(String inParameter, TimeUnit timeUni
     public static String[] getStringArrayParameter(String inParameter) {
         return getConfig().getStringArray(inParameter);
     }
+
+    /**
+     * Get Kitodo data directory.
+     *
+     * @return String
+     */
+    public static String getKitodoDataDirectory() {
+        return getParameter(METADATA_DIRECTORY);
+    }
+
+    /**
+     * Get Kitodo config directory.
+     *
+     * @return String
+     */
+    public static String getKitodoConfigDirectory() {
+        return ConfigCore.getParameter(CONFIG_DIR);
```

Comment:
This could be simplified to `return getParameter(CONFIG_DIR);` or did I miss something?